CC/src/RegistrationTools.cpp

```diff
@@ -81,7 +87,7 @@ void RegistrationTools::FilterTransformation(	const ScaledTransformation& inTran
 				//simpler/faster to ignore this (very) specific case!
 			}
 		}
-		else if (filters & SKIP_RXZ) //keep only the rotation component around Y
```

Comment:
I don't believe you can actually release two rotations axes at the same time. Especially since in the way it is currently implemented, some elements of the matrix set in the first block will be overwritten by the next block. And you'll end with rubbish...

Reply:
Up?